guides/how-to-use-confidential-containers-with-nydus-snapshotter.md

```diff
@@ -0,0 +1,160 @@
+# How to Use Confidential Containers with Nydus Snapshotter
+This document provides an overview on how to run Confidential Containers with [Nydus Snapshotter](https://github.com/containerd/nydus-snapshotter). 
+
+## Introduction
+Confidential Containers(CoCo) protects data in Trusted Execution Environment(TEE) by pulling images in the guest with forked containerd. To optimize resource usage and avoid the need for forking containerd while enabling image pulls within the guest, we employ the Nydus Snapshotter as a proxy plugin. This external plugin for containerd offers two operational modes: one for sharing images on the host with data integrity, and the other for pulling images within the guest. See [Image management proposal](https://github.com/confidential-containers/confidential-containers/issues/137) for detailed design.
```

Comment:
nit: can you break this up into multiple lines? Also, there should be a space before the parentheticals (like this, for example).

Reply:
Sure. @fitzthum Thanks for review.